src/status_im/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -151,14 +151,22 @@
 (defn- page-top
   [{:keys [community-id identical-choices? can-edit-addresses?]}]
   (let [{:keys [name logo color]} (rf/sub [:communities/for-context-tag community-id])
-        confirm-discard-changes   (rn/use-callback
-                                   (fn []
-                                     (if identical-choices?
-                                       (rf/dispatch [:dismiss-modal :addresses-for-permissions])
-                                       (rf/dispatch [:show-bottom-sheet
-                                                     {:content (fn [] [confirm-discard-drawer
-                                                                       community-id])}])))
-                                   [identical-choices?])]
+        {:keys [has-permissions?]} (rf/sub [:community/token-permissions community-id])
+        confirm-discard-changes (rn/use-callback
+                                 (fn []
+                                   (if identical-choices?
+                                     (rf/dispatch [:dismiss-modal :addresses-for-permissions])
+                                     (rf/dispatch [:show-bottom-sheet
+                                                   {:content (fn [] [confirm-discard-drawer
+                                                                     community-id])}])))
+                                 [identical-choices?])
```

Comment:
should not this be tracking `community-id` as well? I take if community-id changes the whole component will be thrown away, but still seems safer to track